src/styles/stars.css

```diff
@@ -0,0 +1,44 @@
+.section {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-bottom: 4px;
+  background-image: url('../images/star.svg');
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars__star:nth-last-child(-n + 5) {
+  margin-bottom: 0;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
```

Comment:
This style does not apply anywhere, because (-n + 0) don't exist, so you can remove it